js/client-teambuilder.js

```diff
@@ -201,9 +209,30 @@
 			this.curTeam.iconCache = '!';
 			Storage.activeSetList = this.curSetList = Storage.unpackTeam(this.curTeam.team);
 			this.curTeamIndex = i;
+			this.currentMoving = -1;
+			this.update();
+		},
+		move: function(i) {
+			this.currentMoving = parseInt(i);
+			this.update();
+		},
+		moveHere: function(i) {
+			i = parseInt(i);
+			if (this.currentMoving < 0) return;
+			if (this.currentMoving === i) {
+				this.currentMoving = -1;
+				this.update();
+				return;
+			}
+			if (!Storage.teams[this.currentMoving]) return;
+			var teamMoving = Storage.teams.splice(this.currentMoving, 1)[0];
+			Storage.teams.splice(i, 0, teamMoving);
+			Storage.saveAllTeams();
```

Comment:
`saveAllTeams` is sort of unnecessary to use here.

Like, it's a decent amount of work on the desktop client, which doesn't support rearranging in the first place.